torch/quantization/fx/utils.py

```diff
@@ -360,7 +360,8 @@ def create_getattr_from_value(module: torch.nn.Module, graph: Graph, prefix: str
     """
     get_new_attr_name = get_new_attr_name_with_prefix(prefix)
     attr_name = get_new_attr_name(module)
-    module.register_buffer(attr_name, torch.tensor(value))
+    device = assert_and_get_unique_device(module)
+    module.register_buffer(attr_name, torch.tensor(value).to(device))
```

Comment:
nit: torch.tensor(value, device=device) would be faster I think since it creates a tensor directly on the device

Reply:
same for other cases

Reply:
ok, will fix